src/copyFile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+// const defaultPath = path.join(__dirname, '../');
+
+const makeFullPath = (pathToAdd) => {
+  return path.join(__dirname, '../', pathToAdd);
+};
+
+function copyFile(souceFilePath, copyFilePath) {
+  const fullSourcePath = souceFilePath.startsWith('.')
+    ? makeFullPath(souceFilePath)
+    : souceFilePath;
+
+  const fileContent = fs.readFileSync(fullSourcePath, 'utf8');
```

Comment:
Most files are not text files encoded in utf-8
```suggestion
  const fileContent = fs.readFileSync(fullSourcePath);
```